aws/resource_aws_emr_instance_group.go

```diff
@@ -55,6 +55,11 @@ func resourceAwsEMRInstanceGroup() *schema.Resource {
 				Optional: true,
 				ForceNew: true,
 			},
+			"bid_price": {
```

Comment:
[Here](https://github.com/terraform-providers/terraform-provider-aws/pull/1438/files#diff-c081c3ba2c4eb51067d67a5e2dd80a85R191), you can see `d.Set("instance_type", group.InstanceType)`, which writes the value returned by the AWS API call, to the state. 
We should use the same approach for the newly added argument, doing some sort of `d.Set("bid_price", group.BidPrice)`